livox_ros_driver/livox_ros_driver/lddc.cpp

```diff
@@ -518,6 +518,21 @@ uint32_t Lddc::PublishImuData(LidarDataQueue *queue, uint32_t packet_num,
   imu_data.linear_acceleration.y = imu->acc_y;
   imu_data.linear_acceleration.z = imu->acc_z;
 
+  // Fill in covariance matrices
+  cur_node_->param("angular_velocity_cov", angular_velocity_cov_, kDefaultAngularVelocityCov_);

```

Comment:
I'm thinking that these covarance matrices should be set on class construction, rather than having to look up the covarance matrix from the ros param server on every IMU message.

Reply:
We're publishing the IMU data at a rate of ~200Hz, so the cost to access the param server is probably non-negligible.

Reply:
Do we expect these covariance values to be updated during runtime?

Reply:
Very good point. For now, we don't want to update the covariances during runtime. In the future, we could use "dynamic reconfigure".

Reply:
I applied your suggestion.